<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linux_Taskmanager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>201</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnProcessPage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Sans Mono</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Process_Manager</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStatsPage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu Sans Mono</family>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>CPU_RAM_USAGE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnWiFiPage">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>WIFI</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>851</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="pageProcess">
     <widget class="QTableWidget" name="tableProcesses">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>811</width>
        <height>371</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>C059</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::PenStyle::SolidLine</enum>
      </property>
      <property name="columnCount">
       <number>5</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Pid</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>State</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Memory</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>User</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnShowAll">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ShowAll</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>30</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineSearch">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnKillProcess">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>KILL</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageStats">
     <widget class="QWidget" name="ramChartContainer" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>431</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="cpuChartContainer" native="true">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>391</width>
        <height>411</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QListWidget" name="listWiFiNetworks">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>691</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelConnectedWiFi">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefreshWiFi">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Sans</family>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1104</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
